src/solver/simulation/sim_calcul_economique.cpp

```diff
@@ -566,28 +567,21 @@ void SIM_RenseignementProblemeHebdo(const Study& study,
 
         for (uint k = 0; k < nbPays; ++k)
         {
-            auto& tsIndex = NumeroChroniquesTireesParPays[numSpace][k];
             auto& area = *(study.areas.byIndex[k]);
             auto& scratchpad = area.scratchpad[numSpace];
             auto& ror = area.hydro.series->ror;
+            auto loadSeries = area.load.series->getCoefficient(year, hourInYear);
```

Comment:
As pointed by @flomnes , we can retrieve the full columns only once et the start and use them (it avoids to repeat the small logic on the index)